examples/vision/image_classification_efficientnet_fine_tuning.py

```diff
@@ -0,0 +1,423 @@
+Resource limit. Perfect compound scaling would assume spatial (memory) and time allowance
+for the computation to grow simultaneously, but OOM may further bottleneck the scaling of
+resolution.
+
+As a result, compound scaling factor is significantly off from
+https://arxiv.org/abs/1905.11946. Hence it is important to understand the compound
+scaling as a rule of thumb that leads to this family of base models, rather than an exact
+optimization scheme. This also justifies that in the keras implementation (detailed
+below), only these 8 models, B0 to B7, are exposed to the user and arbitrary width /
+depth / resolution is not allowed.
+
+## Keras implementation of EfficientNet
+
+An implementation of EfficientNet B0 to B7 has been shipped with tf.keras since TF2.3. To
+use EfficientNetB0 for classifying 1000 classes of images from imagenet, run
+```
+from tensorflow.keras.applications import EfficientNetB0
+model = EfficientNetB0(weights='imagenet')
+```
+
+This model takes input images of shape (224, 224, 3), and the input data should range
+[0,255]. Resizing and normalization are included as part of the model.
+
+Because training EfficientNet on imagenet takes a tremendous amount of resources and
+several techniques that are not a part of the model architecture itself. Hence the Keras
+implementation by default loads pre-trained weights with AutoAugment
+(https://arxiv.org/abs/1805.09501).
+
+For B0 to B7 base models, the input shapes are different. Here is a list of input shape
+expected for each model:
+
+| Base model | resolution|
+|----------------|-----|
+| EfficientNetB0 | 224 |
+| EfficientNetB1 | 240 |
+| EfficientNetB2 | 260 |
+| EfficientNetB3 | 300 |
+| EfficientNetB4 | 380 |
+| EfficientNetB5 | 456 |
+| EfficientNetB6 | 528 |
+| EfficientNetB7 | 600 |
+
+When the use of the model is intended for transfer learning, the Keras implementation
+provides a option to remove the top layers:
+```
+model = EfficientNetB0(include_top=False, weights='imagenet')
+```
+This option excludes the final Dense layer that turns 1280 features on the penultimate
+layer into prediction of the 1000 classes in imagenet. Replacing the top with custom
+layers allows using EfficientNet as a feature extractor and transfers the pretrained
+weights to other tasks.
+
+Another keyword in the model builder worth noticing is `drop_connect_rate` which controls
+the dropout rate responsible for stochastic depth (https://arxiv.org/abs/1603.09382).
+This parameter serves as a toggle for extra regularization in finetuning, but does not
+alter loaded weights.
+
+
+
+## Example: EfficientNetB0 for CIFAR-100.
+
+As an architecture, EfficientNet is capable of a wide range of image classification
+tasks. For example, we will show using pre-trained EfficientNetB0 on CIFAR-100. For
+EfficientNetB0, image size is 224.
+"""
+
+# IMG_SIZE is determined by EfficientNet model choice
+IMG_SIZE = 224
+
+"""
+### prepare
```

Comment:
Fix title (use the `add_example` command to preview your rendered file & notebook)